samza-core/src/test/java/org/apache/samza/operators/TestStreamGraphSpec.java

```diff
@@ -520,4 +434,42 @@ public void testGetTable() {
         new TableSpec("t1", KVSerde.of(new NoOpSerde(), new NoOpSerde()), "", new HashMap<>()));
     assertNotNull(graphSpec.getTable(mockTableDescriptor));
   }
+
+  class MockExpandingSystemDescriptor extends ExpandingSystemDescriptor<Integer, MockExpandingSystemDescriptor> {
```

Comment:
Can you use those other Mock* classes that you added for this?

Reply:
They're in the samza-api module tests, and this is in samza-core module tests. Would prefer to keep them separate.